packages/init/__tests__/sendStarterData.test.ts

```diff
@@ -0,0 +1,419 @@
+import { describe, test, jest, afterEach, expect } from "@jest/globals";
```

Comment:
I think the code of starterData should be splitted into smaller function that can be each tested independently.

Testing a huge function here is good but it's way too hard to read and if something break it will be though to find.

Reply:
If it was going to be a module, used by other people then yes, I would test it independently most so the user could do the set up for the consumer of the function. Here though it's by product of the test requirements.

Ideally we would be testing the entry point, that way dead code is easy to find and eliminate.